CHANGES.rst

```diff
@@ -1,11 +1,38 @@
 CHANGES
 =======
 
-1.2.1 (unreleased)
+1.3.0 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Added Django 1.7 compatibility (Antoine Catton, Rocky Meza, #35)
+- **Backwards Incompatible:** Can't pass the ``next`` URL as a POST parameter
+  - ``next`` should be only passed in the querystring (as a GET parameter)
+
+  If your ``login.html`` template looks like this:
+
+  .. code:: html
+
+    <form method="post" action="{% url 'login' %}">
+        <input type="hidden" name="next" value="{{ next }}">
+        <label>Login: <input type="text" name="login"></label>
+        <!-- ... -->
+
+  You should replace it with:
+
+  .. code:: html
+
+    <form method="post">
+        <label>Login: <input type="text" name="login"></label>
+        <!-- ... -->
+
+  This works because the current URL of the page already contains ``next`` in its querystring.
+
+- **Backwards Incompatible:** ``LoginView.disallow_authenticated`` was changed to ``LoginView.allow_authenticated``
```

Comment:
Why are we doing this as part of django 1.7 support? Why are you silently ignoring disallow_authenticated instead of raising a DeprecationWarning?